README.md

```diff
@@ -111,5 +111,35 @@ icon: https://pbs.twimg.com/profile_images/929088242456190976/xjkS2L-0_400x400.j
 url: https://github.com/ws-skeleton/che-editor-theia/releases/download/untagged-892e01b21d0145207b0f/che-editor-plugin.tar.gz
 ```
 
+### Plugin meta YAML structure:
```

Comment:
I think it makes sense to move this section above.

Reply:
moved